recipes-multimedia/gstreamer/gstreamer1.0_1.16.%.bbappend

```diff
@@ -1,5 +1,7 @@
 FILESEXTRAPATHS_prepend := "${THISDIR}/${BPN}:"
 
-SRC_URI += "file://0001-typefind-min-1k.patch \
-    file://0002-Small-robustness-fixes.patch"
+SRC_URI_append = "\
```

Comment:
Could you please add space between " and / ?

Reply:
done